src/routes/Profile/Profile.tsx

```diff
@@ -37,11 +37,15 @@ const Profile: React.FunctionComponent = () => {
 
   const [copyLinkButtonMode, setCopyLinkButtonMode] = useState<CopyLinkButtonProps['mode']>('copy');
 
-  const fetchBadges = async (seasonId: Season['id'], account: string) => {
+  const fetchBadges = async ( account: string) => {
+    const params = getSDKInitParams(wallet.signer);
+    if (!params) {
+      return getDefaultResponse(season.id);
+    }
     return await getSeasonBadges({
       userId: account,
-      seasonId: seasonId,
-      signer: wallet.signer as Signer,
+      season: season,
+      sbt: new CommunitySBT(params)
```

Comment:
why don't we construct the sbt at the `getSeasonBadges` utility. This way UI keeps UI stuff and 0 knowledge of SDK for this purpose

Reply:
I added a utility to give us Community SBT directly without the params ce1158a161687e6b84a00e7a94627b53bfe1607c. Does this sound ok?

Reply:
sure works for now! thanks!